doc/reference/configuration/configuration_reference.rst

```diff
@@ -4767,6 +4770,36 @@ To learn more about the WAL configuration, check the :ref:`Persistence <configur
     | Default: 16777216
     | Environment variable: TT_WAL_QUEUE_MAX_SIZE
 
+..  _configuration_reference_wal_retention_period:
+
+..  confval:: wal.retention_period
+
+    **Since:** :doc:`3.1.0 </release/3.1.0>` (`Enterprise Edition <https://www.tarantool.io/compare/>`_ only)
+
+    The delay in seconds used to prevent the :ref:`Tarantool garbage collector <configuration_persistence_checkpoint_daemon>` from removing a :ref:`write-ahead log <internals-wal>` file after it has been closed.
+    If a node is restarted, ``wal.retention_period`` counts down from the last modification time of the write-ahead log file.
+
+    The garbage collector doesn't track write-ahead logs that are to be :ref:`relayed <memtx-replication>` to anonymous replicas, such as:
+
+    *   Anonymous replicas added as a part of a cluster configuration (see :ref:`replication.anon <configuration_reference_replication_anon>`).
+    *   CDC (Change Data Capture) that retrieves data using anonymous replication.
+
+    In case of a replica or CDC downtime, the required write-ahead logs can be removed.
+    As a result, such a replica needs to be rebootstrapped.
+    You can use ``wal.retention_period`` to prevent such issues.
+
+    Note that :ref:`wal.cleanup_delay <configuration_reference_wal_cleanup_delay>` option also sets the delay used to prevent the Tarantool garbage collector from removing write-ahead logs.
+    The difference is that the garbage collector doesn't take into account ``wal.cleanup_delay`` if all the nodes in the replica set are up and running, which may lead to the removal of the required write-ahead logs.
+
+    ..  NOTE::
+
+        The minimum vclock value can be found using :ref:`box.info.gc().wal_retention_vclock <box_info_gc>`.
```

Comment:
This one is vclock of the oldes xlog, which is protected from GC, by `wal_retention_period`. IMHO, now it's non obvious what this vclock shows

Reply:
thanks, will extend the description